### PR TITLE
Fixes T348416 Change UploadedItems.js component to stateless functional components

### PR DESCRIPTION
**Pull request by Ree-m**
## Files Changed
- UploadedItems.js

## Why?
- UploadedItems.js is currently based on class component and uses state. However, this module can be easily migrated to a stateless functional component.

### CODE REVIEW

File: components/UploadedItems.js [hunk 1]
```diff
@@ -1,57 +1,47 @@
...
-  componentDidMount() {
-    this.fetchData();
-  }
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        const response = await fetch(url);
+        const responseJson = await response.json();
+        const booksCount = await responseJson.response.numFound;
+        console.log("booksCount", booksCount);
```
Comment (Parthiv-M) [inline components/UploadedItems.js:14]: This logging statement can be removed I believe

**Review by Parthiv-M — CHANGES_REQUESTED** [review]
Code looks alright to me
- Please restore the `.env.example` file

File: components/UploadedItems.js (hunk 1 above)
Comment (Ree-m) [reply components/UploadedItems.js]: Noted! I removed the console log and added the .env.example file back.

**Review by Parthiv-M — APPROVED** [review]
LGTM

**Comment by Parthiv-M** [comment]
Closing, as the component is no longer used

**Pull request closed without merging**